### PR TITLE
Adding hasura go-client

### PR DESCRIPTION
**Pull request by rajdas98**
Closes #<issue number>

## Description

- Adding Hasura client with Mutation, Subscription and query support

### CODE REVIEW

File: src/content/code/language-support/go/client/go-graphql-client.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+name: graphql
```
Comment (mikeesto) [inline src/content/code/language-support/go/client/go-graphql-client.md:2]: Should this be `go-graphql-client` or similar?

Comment (rajdas98) [reply src/content/code/language-support/go/client/go-graphql-client.md]: yes @mikeesto , Made the change.

**Comment by orta** [comment]
Are you sure you want to skip on adding some markdown to describe the usage?

**Comment by rajdas98** [comment]
Yes, @orta. It's an early-stage project, nothing to add much but this is the only project who support subscription in go client. If it is mandatory, I can add it.

**Comment by orta** [comment]
That's fine 👍🏻

**Pull request merged**